week-3/Homework/mandatory/4-eligible-students.js

```diff
@@ -7,8 +7,11 @@
   - Returns an array containing only the names of the who have attended AT LEAST 8 classes
 */
 
-function eligibleStudents() {
-
+function eligibleStudents(arr) {
+  let students = arr
+    .filter((x) => x[1] >= 8)
+    .map((x) => x.splice(0, 1).join(" "));
```

Comment:
`splice` here will change the array we were passed, which may cause problems for the caller. Can you make this do the same thing, except not change `arr`?